### PR TITLE
Fixes issue on Safari where form texts jogs on input.  

### PR DESCRIPTION
**Pull request by DaveAMA**
Description:

On Safari when a user has form autofill enabled, the placeholder text in a form field jogs down a few pixels when selected:
jump-unresolved

I have taken a look at forms on membership, registries, gatekeeper, and my_ama on both Safari and Chrome and it looks like all forms are working well:
jump-resolved

To resolve the text-jogging issue only `input` needs `height: auto` applied to it, but this resulted in the form field being 1 or 2 pixels higher than our drop-down menus. Applying the height to both elements maintains height consistency across the form.
~~~~
    & input{
      height: auto;
    }
    & select{
      height: auto;
    }
~~~~

### CODE REVIEW

**Comment by DaveAMA** [comment]
💩

File: app/assets/stylesheets/layout_components/forms.scss [hunk 1]
```diff
@@ -446,6 +446,14 @@ input[type="url"]:-ms-input-placeholder{
         margin-bottom: $base-margin;
       }
     }
+
+    /* Prevents jogging of an input's placeholder text due to safari's form autofill. */
+    & input{
```
Comment (kaytcampbell) [inline app/assets/stylesheets/layout_components/forms.scss]: I don't think you need the & here

& input {
}

and 

input {
}

Would be the same thing. I also wonder if we're better off putting this on all form tags? So like 
form input,
form select{
  height: auto;
}

**Comment by DaveAMA** [comment]
@kaytcampbell I've updated the form selectors to increase the scope, they look good on all the forms I've tested. 👋

**Comment by DaveAMA** [comment]
👋

**Comment by kaytcampbell** [comment]
:art: :art: :art:

**Pull request merged**